include/ajax.tickets.php

```diff
@@ -641,6 +641,10 @@ function editField($tid, $fid) {
                         if (strlen($clean) > 200)
                              $clean = Format::truncate($clean, 200);
                         break;
+                    case $field instanceof BooleanField:
+                        $clean = $field->getClean();
+                        $clean = $field->toString($clean) ?: '&mdash;' . __('Empty') .  '&mdash;';
```

Comment:
It looks like $field->toString($clean) will always return Yes or No no matter what you pass into toString, so I don't think you need to put the 'empty' part in there. I tried manually passing in random strings, numbers, and quotes, and every time it would return either Yes or No. Not sure if we should just leave it there to be absolutely sure though. It doesn't hurt anything.

Reply:
Looking at it now, I think I can remove it because if you look at [this code snippet](https://github.com/osTicket/osTicket/blob/develop-next/include/ajax.tickets.php#L650-L654) we return "empty" if there is no `$clean`.